helita/sim/bifrost.py

```diff
@@ -77,6 +77,7 @@ def __init__(self, file_root, snap=None, meshfile=None, fdir='.',
         self.lowbus = lowbus
         self.numThreads = numThreads
         self.file_root = os.path.join(self.fdir, file_root)
+        self.root_name = file_root
```

Comment:
Since this is not used in BifrostData, perhaps you can just overload this in the definition of EbysusData?

Reply:
This is used in other codes such as visualization tools from mikolaj. I think it is nice to be able to get the root name too for other proposes.